mtags/src/main/scala-3/scala/meta/internal/pc/PcInlineValueProviderImpl.scala

```diff
@@ -55,13 +53,44 @@ final class PcInlineValueProviderImpl(
         defPos.toLsp,
         adjustRhs(definition.tree.rhs.sourcePos),
         RangeOffset(defPos.start, defPos.end),
+        definitionRequiresBrackets(definition.tree.rhs)(using newctx),
         deleteDefinition,
       )
 
       (defEdit, refsEdits)
     end for
   end defAndRefs
 
+  // format: off
```

Comment:
Please don't use `format: off` we can introduce settings if needed, but I would rather avoid it if possible.